src/ImageUpload.js

```diff
@@ -48,23 +94,78 @@ function ImageUpload({ username }) {
             setProgress(0);
             setCaption("");
             setImage(null);
+            setUpload(false);
+            setFile(null);
           });
       }
     );
   };
 
   return (
-    <div className="imageupload">
-      <progress className="imageupload__progress" value={progress} max="100" />
-      <Input
+
+    //Upload Form UI changed
+    <Card className="imageupload">
+      <CardContent>
+        <Typography variant="h5" style={{textAlign:'center',marginBottom:'10px'}}>Add Post</Typography>
```

Comment:
Mention a className here and add styling in useStyle const. It's a good practise.

Reply:
Okay making the changes!